### PR TITLE
Add Vietnamese translation

### PR DESCRIPTION
**Pull request by leduyquang753**
I translated the phrases to Vietnamese based on the file en.yml.

### CODE REVIEW

File: webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java [hunk 1]
```diff
@@ -321,17 +321,26 @@ private static PdfReader createPdf(String globalTitle, Date creationDate, Scramb
...
+            //            Element.ALIGN_RIGHT, new Phrase(pageN + "/" + pr.getNumberOfPages()),
+            //            rect.getRight(), rect.getTop(), 0);
+            //}
 
             // Footer
-            String generatedBy = "Generated by " + Utils.getProjectName() + "-" + Utils.getVersion();
-            ColumnText.showTextAligned(cb,
-                    Element.ALIGN_CENTER, new Phrase(generatedBy),
-                    (pageSize.getLeft() + pageSize.getRight()) / 2, pageSize.getBottom() + 40, 0);
+			switch (pr.getNumberOfPages % 2) {
```
Comment (jfly) [inline webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java]: I'm interested in learning more about what you're doing here, but can you do this in a separate PR? Is this related to the Vietnamese translation somehow?

File: webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java [hunk 2]
```diff
@@ -102,8 +102,8 @@
     private static BaseFont monoFont, sansSerifFont;
     static {
         try {
-            monoFont = BaseFont.createFont("fonts/LiberationMono-Regular.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
-            sansSerifFont = BaseFont.createFont("fonts/wqy-microhei.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
+            monoFont = BaseFont.createFont("fonts/segoeui.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
+            sansSerifFont = BaseFont.createFont("fonts/segoeui.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
```
Comment (jfly) [inline webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java]: This is concerning to me: is `segoeui` a monospaced font? Does it have support for all the characters we use in all our other translations?

Does `wqy-microhei` not have the Vietnamese characters you need? If we need to find a font that supports every language we do, this might be useful: https://github.com/thewca/tnoodle/pull/246#issuecomment-351934232. Alternatively, we could implement support for using different fonts per language.

**Comment by jfly** [comment]
Also, please include your name and the name of the WCA Delegate who approved this translation!

File: webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java (hunk 1 above)
Comment (leduyquang753) [reply webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java]: I changed the way the number of pages are being printed. But this is for myself only. I thought the pull request only contains my Vietnamese translation... :/ Please don't notice my changes to the code.

**Comment by leduyquang753** [comment]
I'm trying to contact to [Trịnh Nguyên Anh](mailto:nguyenanh86@gmail.com) for the approval of the translation. Thank you for prompting me to do that!

**Comment by jfly** [comment]
I'm closing this in favor of https://github.com/thewca/tnoodle/pull/278. Please feel free to send in a PR with any changes required to get TNoodle compiling on Windows!
